inlong-sort-standalone/bin/sort-start.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+#
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+source /etc/profile
+cd "$(dirname "$0")"/../
+chmod 777 bin/flume-ng
+nohup bin/flume-ng agent --conf conf/ > sort.log 2>&1 &
```

Comment:
it's better to remove the `flume` word if this module is only for `sort-standalone`.

Reply:
change flume-ng to sort-standalone.